### ISSUE TITLE
Nothing is displayed.

### ISSUE THREAD

**Issue by m-kru** [folke/polydock#4]
I have tried running it from release zip, however nothing is displayed. In the terminal I get following log:
```bash
Gjs-Message: 17:45:55.345: JS LOG: [app] startup
Gjs-Message: 17:45:55.367: JS LOG: [app] activate
Gjs-Message: 17:45:55.367: JS LOG: [settings] /tmp/polydock-bin-2.0.2/bin/config/settings.ini
Gjs-Message: 17:45:55.368: JS LOG: [theme] /tmp/polydock-bin-2.0.2/bin/config/themes/default.css
Gjs-Message: 17:45:55.369: JS LOG: Alpha Visuals: true
Gjs-Message: 17:45:55.369: JS LOG: Composited: true
Gjs-Message: 17:45:55.376: JS LOG: [update] + constructor
Gjs-Message: 17:45:55.387: JS LOG: + 2 brave-browser
Gjs-Message: 17:45:55.388: JS LOG: [icon] /usr/share/icons/hicolor/scalable/apps/nemo.svg
Gjs-Message: 17:45:55.391: JS LOG: + 2 nemo
Gjs-Message: 17:45:55.392: JS LOG: [icon] /usr/share/icons/hicolor/scalable/apps/com.gexperts.Tilix.svg
Gjs-Message: 17:45:55.393: JS LOG: + 2 tilix
Gjs-Message: 17:45:55.394: JS LOG: [icon] /usr/share/icons/hicolor/scalable/apps/com.gexperts.Tilix.svg
Gjs-Message: 17:45:55.394: JS LOG: + 2 tilix
Gjs-Message: 17:45:55.395: JS LOG: + 2 polybar
Gjs-Message: 17:45:55.396: JS LOG: [update] - constructor [4]
Gjs-Message: 17:45:55.396: JS LOG: [toobar] show
Gjs-Message: 17:45:55.434: JS LOG: [update] + window-opened
Gjs-Message: 17:45:55.436: JS LOG: + 3 polydock
Gjs-Message: 17:45:55.437: JS LOG: [update] - window-opened [5]
Gjs-Message: 17:45:55.437: JS LOG: [toobar] show
Gjs-Message: 17:45:55.439: JS LOG: [update] + workspace-changed
Gjs-Message: 17:45:55.441: JS LOG: [update] - workspace-changed [5]
Gjs-Message: 17:45:55.441: JS LOG: [toobar] show
Gjs-Message: 17:45:55.458: JS LOG: [update] + window-closed
Gjs-Message: 17:45:55.458: JS LOG: - 3 polydock
Gjs-Message: 17:45:55.459: JS LOG: [update] - window-closed [4]
Gjs-Message: 17:45:55.459: JS LOG: [toobar] show
```

**Comment by folke** [comment]
What WM are you using?

The default location for the dock is top center. Could there be another bar or something over the dock?

For example, I start polydock after polybar and that seems to make sure it's dispayed on top of polybar.

**Comment by m-kru** [comment]
OpenBox. I do not use any bar.

**Comment by folke** [comment]
The problem seems to be that OpenBox manages the Polydock window, even though it is a `dock`. Polydock then sees it's own window overlap with itself and automatically hides.

I'll release a fix later today.

**Comment by folke** [comment]
This should be fixed in https://github.com/folke/polydock/releases/tag/2.0.3

Let me know if you would still experience any issues!

**Comment by m-kru** [comment]
Ok, it works. Any way to force it to never hide?

**Comment by folke** [comment]
I just pushed a new release where you can disable `autoHide`:

```ini
[behavior]
autoHide=false
```

**Comment by stjiub** [comment]
I am having the same issue on the latest release while running on OpenBox. Polydock is running, but not visible. I use polybar, but even with it not running I cannot see polydock. I was using the polywins module before with polybar and if I ran polydock I could see its name pop up in the polywins dock and if right clicked on it I could select somewhere else on the desktop to "place" it. This made it visible, but also full screen across my 3 monitors... So it is technically running, but not sure where OpenBox is hiding it. I disabled "autoHide" as well.
